### ISSUE TITLE
[Question] Incrementing via Commit Message

### ISSUE THREAD

**Issue by pushasha** [GitTools/GitVersion#777]
As a user, if I do the following, what is the expected outcome if I run GitVersion from the command line (assume default settings)?

1. Make an initial commit to master in a bare repository, and tag that commit as `1.0.0`
2. Make a commit directly to master with nothing semver-related in commit message (note: by default, I know this increments the patch number after a tag, so I use it as a sanity check)
3. Make commit directly to master, with `+semver: minor` included in commit message
4. Make commit directly to master again, also with `+semver: minor` included in commit message

After reading [the documentation](http://gitversion.readthedocs.org/en/latest/more-info/version-increments/#commit-messages), I was under the impression that the calculated semantic version would be `1.2.0`, but that doesn't seem to be the case (I'm getting `1.1.0`). Is only the most significant (i.e. "largest") increment value for _all_ commits since a tag taken and applied to the version? Or should each commit with a relevant message be evaluated separately and "added" to the version? 

In other words, if I have a commit tagged as `1.0.0`, then make fifty commits, each with a `+semver:minor` commit message, should the version be `1.50.0` or `1.1.0`?

Apologies for asking such a simple question -- I'm just worried that I might have a fundamental misunderstanding of what GitVersion is meant to do.

**Comment by pushasha** [comment]
Issue was answered on Gitter -- This _is_ the expected behavior. For anyone else looking, the explanation is here: https://github.com/GitTools/GitVersion/issues/708#issuecomment-154488513
